examples/ale_regression_boston.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>I would have put _feature effects on the model_ to emphasise this is a global method (so applies to <em>any</em> prediction not specified predictions (_the predictions_))</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>